### ISSUE TITLE
textLength and lengthAdjust text attributes are not supported

### ISSUE THREAD

**Issue by cellofellow** [Kozea/CairoSVG#159]
These attributes allow "squishing" or "stretching" of a text element. I need this to be able to fit arbitrary texts into fixed sized fields for print.

[W3C](https://www.w3.org/TR/SVG/text.html#TextElementTextLengthAttribute) has specs for these features.

**Comment by cellofellow** [comment]
Unfortunately that other Python-usable SVG library (rsvg) doesn't support this either. I tried using Inkscape from a child process but this is proving awkward. In fact, the only SVG programs I've used that properly support this feature are Firefox and Inkscape.

I did find a node library that seems to have some support for it. [svg-to-pdfkit](https://github.com/alafr/SVG-to-PDFKit/blob/master/source.js#L1980)

I'd be glad to help if needed.
